web-services-guide/ent-and-web-services/transfer-server.md

```diff
@@ -198,6 +198,49 @@ Content Station 9.5 uses Deflate compression when remote users are saving (uploa
 articles. This feature becomes affective when in the LogOnResponse the ‘IsRemoteUser’ feature is set to ‘true' and the 
 ‘AcceptsCompressions’ feature lists the ‘deflate’ technology.
 
+## Signed transfer server URLs \[since 10.7\]
+
+Since Enterprise Server version 10.7.0 it is possible to request signed download links from the server. This
+will allow clients to download files from the transfer server without knowing the ticket and fileguid. The signed url can 
+only be generated with a client that has a valid session ticket and the token is only valid for a specific amount of 
+time for only the file it was generated for. This feature is useful when you want to get a download link and pass it to a 
+different application (for example for downloading). 
+
+To request a signed URL from the server you can add the following parameters to the server URL that is used to call 
```

Comment:
server URL that is used to call a service on Enterprise Server => web service entry point